CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
-## 1.1.6-dev
+## 2.0.0
```

Comment:
I had hoped we would publish as `1.2.0` with a new argument and deprecate the old ones, migrate the test runner, then bump the version...

If we are jumping straight to the breaking release can we also give a type on `sdkRoot`?
https://github.com/dart-lang/source_map_stack_trace/pull/9

Reply:
Sure - I don't think there is much advantage to doing the non-breaking release